frontend/tests/e2e/alltests.js

```diff
@@ -32,8 +33,8 @@ test('Test search', async t => {
     await t.typeText('#user-search', 'testcafe')
         .click('#search-button')
         .expect(Selector('html').textContent)
-        // This is the ID of the testcafe user.
-        .contains(TEST_CAFE_USER_ID);
+        // Search with Username not with ID anymore
+        .contains(TEST_CAFE_USERNAME);
```

Comment:
The comment "Search with Username not with ID anymore" won't be helpful in the near future because future maintainers won't know we ever searched by ID. 

"Search for the username" would be fine, or better yet no comment at all. The original comment "This is the ID of the testcafe user" was written before the TestCafe user ID was turned into a named constant (TEST_CAFE_USER_ID), but now that it is a constant the code is self-documenting.